### PR TITLE
fix: Add missing documentation for RTCDTMFSender.canInsertDTMF

### LINKED ISSUE #33951: RTCDTMFSender.canInsertDTMF missing
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/RTCDTMFSender

### What specific section or headline is this issue about?

_No response_

### What information was incorrect, unhelpful, or incomplete?

According to the WebRTC specification ( https://github.com/w3c/webrtc-pc ), the `RTCDTMFSender` interface should have a `canInsertDTMF` attribute. However, I can't find any mention of it in this MDN page.

### What did you expect to see?

Here's an example of what I expected to see:

```md
## Instance properties

- `RTCDTMFSender.canInsertDTMF`
  - A boolean value which is `true` if the `RTCDTMFSender` is capable of sending DTMF tones, or `false` if it is not.
```

### Do you have any supporting links, references, or citations?

https://w3c.github.io/webrtc-pc/#dom-rtcdtmfsender-caninsertdtmf

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/api/rtcdtmfsender`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/RTCDTMFSender
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/rtcdtmfsender/index.md
* Last commit: https://github.com/mdn/content/commit/50ed08d7b506c19b7d073b05ea1e02a15f276878
* Document last modified: 2023-02-19T09:57:47.000Z

</details>

### PR DESCRIPTION
**Pull request by bc-lee**
Fixes #33951

### CODE REVIEW

File: files/en-us/web/api/rtcdtmfsender/caninsertdtmf/index.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+title: "RTCDTMFSender: canInsertDTMF property"
+short-title: canInsertDTMF
+slug: Web/API/RTCDTMFSender/caninsertdtmf
+page-type: web-api-instance-property
+browser-compat: api.RTCDTMFSender.canInsertDTMF
+---
+
+{{APIRef("WebRTC")}}
+
+The {{domxref("RTCDTMFSender")}} interface's `canInsertDTMF` property is a read-only boolean value which indicates whether the `RTCDTMFSender` is capable of sending DTMF tones over the {{domxref("RTCPeerConnection")}}.
```
Comment (skyclouds2001) [inline files/en-us/web/api/rtcdtmfsender/caninsertdtmf/index.md:11]:
```suggestion
The **`canInsertDTMF`** read-only property of the  {{domxref("RTCDTMFSender")}} interface returns a boolean value which indicates whether the `RTCDTMFSender` is capable of sending DTMF tones over the {{domxref("RTCPeerConnection")}}.
```

**Comment by bc-lee** [comment]
Hi, is there anything required to land this PR? Thanks.

File: files/en-us/web/api/rtcdtmfsender/caninsertdtmf/index.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+---
+title: "RTCDTMFSender: canInsertDTMF property"
+short-title: canInsertDTMF
+slug: Web/API/RTCDTMFSender/caninsertdtmf
```
Comment (hamishwillee) [inline files/en-us/web/api/rtcdtmfsender/caninsertdtmf/index.md:4]:
```suggestion
slug: Web/API/RTCDTMFSender/canInsertDTMF
```

**Review by hamishwillee — APPROVED** [review]
Thanks very much. This is good. FYI, it can take some time to review PRs as the team with approver rights is relatively small. In this case @Elchi3 was assigned, and he is extremely overloaded.

**Comment by hamishwillee** [comment]
PS, thanks for review @skyclouds2001 !!!

**Review by illuminated-shade — COMMENTED** [review]
files/en-us/web/api/rtcdtmfsender/caninsertdtmf/index.md

**Comment by illuminated-shade** [comment]
gh pr checkout 33952

**Pull request merged**